docs/interpreter/python.md

```diff
@@ -232,6 +232,67 @@ SELECT * FROM rates WHERE age < 40
 Otherwise it can be referred to as `%python.sql`
 
 
+## IPython Support
+
+IPython is more powerful than the default python interpreter with extra functionality. You can use IPython2 or IPython3 which depends on which python you set `zeppelin.python`.
+
+   **Pre-requests**
+   
+    - Jupyter `pip install jupyter`
+    - IPython `pip install IPython`
+    - ipykernel `pip install ipykernel`
+    - grpcio `pip install grpcio`
+
+Besides the basic function of python interpreter. You can use IPython as you use it in Jupyter Notebook.
```

Comment:
It doesn't work, it needs some javascripts, seems it depends on jupyter notebook. I need to investigate more. 

```
[IPKernelApp] WARNING | Widget Javascript not detected.  It may not be installed properly. Did you enable the widgetsnbextension? If not, then run "jupyter nbextension enable --py --sys-prefix widgetsnbextension"
```